.env.example

```
@@ -71,4 +96,17 @@ APPRISE_SIZE_UNIT=Auto
+# Show uploaded file list in the UI (true/false, default: false)
+SHOW_FILE_LIST=false
+
+# Maximum client retries for chunk upload failures (integer, default: 5)
+CLIENT_MAX_RETRIES=5
+
+# Demo mode disables persistence for uploads (true/false, default: false)
+DEMO_MODE=false
+
+# Internal/testing toggles for cleanup schedulers (leave false in normal use)
+DISABLE_BATCH_CLEANUP=false
+DISABLE_SECURITY_CLEANUP=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)